### ISSUE TITLE
Error width vite

### ISSUE THREAD

**Issue by ehernandezlabelgrup** [xnimorz/masked-input#56]
Hi community, we are trying to use the react-maskinput package and it gives us an error when running the project. Our base is vite, we don't use cra. In other projects we have used cra and the react-maskinput package and it works well, but with vite it gives us the following error: [ERROR] [plugin vite:dep-scan] Failed to resolve entry for package "react-maskinput". The package may have incorrect main/module/exports specified in its package.json. Any ideas on how to solve it?

**Comment by xnimorz** [comment]
Hey, @ehernandezlabelgrup 

Can you try to fetch react-maskinput@3.2.0?
I believe I hit a bug with 3.2.1 and I will work on a fix soon. 
Meanwhile, I'd recommend using 3.2.0

**Comment by xnimorz** [comment]
Actually, I changed the latest version to 3.2.0 back, so it should be fine now.

**Comment by seravifer** [comment]
Any update @xnimorz?

**Comment by xnimorz** [comment]
Version 3.2.0 has no issue and I believe it's marked as a latest atm

**Comment by seravifer** [comment]
You are right, it works in Vite with the latest version without a problem if it is imported directly. If we look at the build, we see that Vite processes it correctly:
```
import __vite__cjsImport11_reactMaskinput from "/node_modules/.vite/deps/react-maskinput.js?v=86502b3b";
const MaskInput = __vite__cjsImport11_reactMaskinput.__esModule ? __vite__cjsImport11_reactMaskinput.default : __vite__cjsImport11_reactMaskinput;
```

But in my case, it is an external library that imports `react-maskinput` and `esbuild` in this case does not process it correctly:
`var import_react_maskinput = __toESM(require_lib2(), 1);`
and React fails because `import_react_maskinput` is `{default : {default: ƒ}}` and not the component it self.

From what I have been able to read, it is a limitation of [esbuild](https://esbuild.github.io/content-types/#default-interop) and the behavior will not be modified.

The only thing I have seen to make it work is to use this library: https://github.com/vite-plugin/vite-plugin-commonjs

**Comment by xnimorz** [comment]
I see, I'll try to take a look at issue in upcoming few weeks. 
There are many things which should be done for this lib
